node_modules/oae-core/upload/js/upload.js

```diff
@@ -35,9 +35,6 @@ define(['jquery', 'oae.core', 'jquery.fileupload', 'jquery.iframe-transport', 'j
         // will be used in the event communication between this widget and the `setpermissions` widget.
         var setPermissionsId = oae.api.util.generateId();
 
-        // IE9 and below don't support XHR file uploads and we fall back to iframe transport
-        var useIframeTransport = !$.support.xhrFileUpload && !$.support.xhrFormDataFileUpload;
```

Comment:
We still reference this variable elsewhere in this file - those references should be updated too.